src/widgets/modals/SearchForm.js

```diff
@@ -31,7 +31,7 @@ import {
 } from '../../selectors/dispensary';
 
 const SearchListItemColumnComponent = ({ value, type }) => {
-  const valueText = type === 'date' ? value?.toDateString() ?? '' : value;
+  const valueText = type === 'date' ? value?.toDateString() ?? generalStrings.not_available : value;
```

Comment:
this would make a good candidate for a helper utility method

Reply:
Yeah, I'm thinking in the near future it would be good to have all "parsers" in one utility scope. AFAICS, the main use cases are converting from database format and interface format (and also from incoming sync to database format), so something like:

```
const date = new Date();
// Interface wants to display date
const dateString = parsers.dateToString(date);
....
// Database wants to save date
const dateObject = parsers.stringToDate(dateString)
```

Ideal would be to combine all the existing logic for handling int/floats/double conversions, object/JSON conversions etc. into one `parser` namespace (maybe even a static class).